### ISSUE TITLE
/rc reload breaks plugin

### ISSUE THREAD

**Issue by TheDeafCreeper** [Aelysium-Group/rusty-connector#28]
this.proxyWhitelist back at it again.

```
> rc reload
[23:45:23 INFO] [rustyconnector-velocity]: Reloading the proxy...
[23:45:23 INFO]: Building config.yml...
[23:45:23 INFO]: Finished building config.yml
[23:45:23 ERROR] [rustyconnector-velocity]: Cannot invoke "java.lang.ref.WeakReference.clear()" because "this.proxyWhitelist" is null
java.lang.NullPointerException: Cannot invoke "java.lang.ref.WeakReference.clear()" because "this.proxyWhitelist" is null
        at group.aelysium.rustyconnector.plugin.velocity.lib.whitelist.WhitelistService.kill(WhitelistService.java:71) ~[?:?]
        at group.aelysium.rustyconnector.core.lib.serviceable.ServiceHandler.lambda$killAll$0(ServiceHandler.java:34) ~[?:?]
        at java.util.HashMap.forEach(HashMap.java:1421) ~[?:?]
        at group.aelysium.rustyconnector.core.lib.serviceable.ServiceHandler.killAll(ServiceHandler.java:34) ~[?:?]
        at group.aelysium.rustyconnector.plugin.velocity.central.Processor.kill(Processor.java:48) ~[?:?]
        at group.aelysium.rustyconnector.plugin.velocity.central.VelocityAPI.reloadServices(VelocityAPI.java:98) ~[?:?]
        at group.aelysium.rustyconnector.plugin.velocity.central.CommandRusty.lambda$create$15(CommandRusty.java:221) ~[?:?]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:262) ~[proxy.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at com.velocitypowered.proxy.command.VelocityCommandManager.executeImmediately0(VelocityCommandManager.java:226) ~[proxy.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at com.velocitypowered.proxy.command.VelocityCommandManager.lambda$executeAsync$0(VelocityCommandManager.java:253) ~[proxy.jar:3.2.0-SNAPSHOT (git-bda1430d-b259)]
        at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
        at java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:833) [?:?]
```

Running `/rc reload` starts the reload process, but when it tries to reload the whitelist it fails (probably assuming if there's no whitelist set), causing other parts of the plugin (such as families) to fail to load as well.

**Comment by nathan-i-martin** [comment]
/rc reload has actually been pretty neglected as of late so I'm not surprise it's broken.
Will plan on fixing it for v0.6.1

**Comment by nathan-i-martin** [comment]
Fixed as of 3c832db1dfa742e4bfddb18d45398b3dd6882607
